lib/galaxy/tool_util/verify/interactor.py

```diff
@@ -1547,23 +1544,29 @@ def register_exception(e: Exception):
             # Legacy - fall back on ordered data list access if data_list is
             # just a list (case with twill variant or if output changes its
             # name).
-            if hasattr(data_list, "values"):
-                output_data = list(data_list.values())[output_index]
-            else:
-                output_data = data_list[len(data_list) - len(testdef.outputs) + output_index]
-        assert output_data is not None
-        try:
-            galaxy_interactor.verify_output(
-                history,
-                jobs,
-                output_data,
-                output_testdef=output_testdef,
-                tool_id=job["tool_id"],
-                maxseconds=maxseconds,
-                tool_version=testdef.tool_version,
-            )
-        except Exception as e:
-            register_exception(e)
+            try:
+                if hasattr(data_list, "values"):
+                    output_data = list(data_list.values())[output_index]
+                else:
+                    output_data = data_list[len(data_list) - len(testdef.outputs) + output_index]
+            except IndexError:
+                pass
```

Comment:
That seems weird, can you add a comment here ? I think that's also why the framework test failed.

Reply:
You are right. How about https://github.com/galaxyproject/galaxy/pull/16094/commits/6ee5b09f95b2fbd06dc59675d72d1dd68dd9431d? Hope this makes it clearer. The comment would have been more or less the content of the exception we are recording here.